src/Components/Footer/Footer.scss

```diff
@@ -1,18 +1,21 @@
 .Footer {
-    position: fixed;
+    // position: fixed;
```

Comment:
Is this commented out accidentally? 1. Footer is not sticky without this line of code 2. If we think that some property is not needed, let's remove it, instead of commenting.